### ISSUE TITLE
Add Reproducible Paper Guidelines

### ISSUE THREAD

**Issue by mkuzak** [esciencecenter-digital-skills/Lesson-FAIR-Data-Climate#44]
Association of Geographic Information Laboratories in Europe has developed Reproducible Paper Guidelines (https://osf.io/numa5/). Do you think it should be added to this lesson?

**Comment by Peter9192** [comment]
related to #20

**Comment by MpaulaL** [comment]
I think it would be nice to reflect the importance of FAIR in reproducibility. I would add this in the introduction of why FAIR? At the moment I don't see much about reproducibility in there.

**Comment by Peter9192** [comment]
Yes great point! I was thinking that we might wanna make the lesson focus more or "reproducible climate science" rather than just "FAIR climate data". Would that make sense?
